heltour/tournament/views.py

```diff
@@ -621,14 +624,35 @@ def view(self, post=False):
             reg_season = Season.objects.filter(league=self.league, registration_open=True).order_by('-start_date').first()
         if reg_season is None:
             return self.render('tournament/registration_closed.html', {})
-
         if post:
             form = RegistrationForm(self.request.POST, season=reg_season)
             if form.is_valid():
                 with reversion.create_revision():
                     reversion.set_comment('Submitted registration.')
                     form.save()
-                self.request.session['reg_email'] = form.cleaned_data['email']
+                username = form.cleaned_data['lichess_username']
+                email = form.cleaned_data['email']
+                reg = {
```

Comment:
I believe you can replace these variables with `form.instance` and use that in place of `reg`.

Reply:
Looks like `email` would be used in two places. If there were more I'd say definitely use the variable; less I'd say don't; for two I have no preference.

Reply:
In case it wasn't clear what I meant here, `form.instance` is a `Registration` object that already has `season`, `lichess_username`, and `email` fields. You can use that rather than constructing your own `reg` object.

Reply:
ah, yeah I misunderstood. Will fix it soon.